### ISSUE TITLE
Build as static library with the CocoaPods 1.5.0

### ISSUE THREAD

**Issue by tdksk** [xmartlabs/XLPagerTabStrip#569]
## Goals

Use static libraries to reduce the launch time on a app

## Expected Results

XLPagerTabStrip would build

## Actual Results
```
Unable to run command 'StripNIB ButtonCell.nib' - this target might include its own product.
Unable to run command 'StripNIB ButtonCell.nib' - this target might include its own product.
Unable to run command 'StripNIB ButtonCell~iphone.nib' - this target might include its own product.
Unable to run command 'StripNIB ButtonCell~iphone.nib' - this target might include its own product.
Unable to run command 'StripNIB ButtonCell~ipad.nib' - this target might include its own product.
Unable to run command 'StripNIB ButtonCell~ipad.nib' - this target might include its own product.
```

## Steps to Reproduce

1. Create a Gemfile:

```
source 'https://rubygems.org'
gem 'cocoapods', '1.5.0'
```

2. Change your Podfile to use

```
use_frameworks!(false)
pod 'XLPagerTabStrip'
```

3. Run `bundle install && bundle exec pod install`
4. Try to build the app

## Version of XLPagerTabStrip and Tooling

XLPagerTabStrip version: 8.0.1
CocoaPods version: 1.5.0
Xcode version: 9.2
iOS version: 11.2

**Comment by andwun** [comment]
I ran into this issue with version 8.0.1, as well.
Please note that it seems to be fixed on `master`.
A `git bisect` indicated that `4fe566f187b7e2c7c883a438bc6ab701a81ebbb3` is the commit that fixes the issue for me.

## Toolchain Versions

CocoaPods version: 1.5.0
Xcode version: 9.3
iOS version: 11.3

**Comment by iafine** [comment]
I also found this problem.

### Toolchain Versions

CocoaPods Version: 1.5.2
Xcode Version: 9.4
iOS Version: 11.4
XLPagerTabStrip Version: 8.0.1

### Podfile

```
# use_frameworks!
...
  pod 'XLPagerTabStrip'
...
```
This error will occur if you comment this code `use_frameworks!`

It's like to a bug!

**Comment by tdksk** [comment]
This issue has fixed after the version [8.1.0](https://github.com/xmartlabs/XLPagerTabStrip/releases/tag/8.1.0). Thanks!
